src/App.js

```diff
@@ -18,11 +18,23 @@ import Responsive from 'react-responsive-decorator';
 import './App.css';
 import './style.css';
 
+const getTimelineCookie = () => {
+  const cookies = document.cookie.split(';');
```

Comment:
We can use external lib here to help us parse the cookie (https://www.npmjs.com/package/js-cookie)

Reply:
Also, let's use a more comprehensive name for this cookie entry, something like `timeline_experiment`.